src/javascript/app/pages/trade/tradepage.js

```diff
@@ -77,6 +78,17 @@ const TradePage = (() => {
         TradingAnalysis.bindAnalysisTabEvent();
 
         ViewPopup.viewButtonOnClick('#contract_confirmation_container');
+
+        displayBanner();
+    };
+
+    const displayBanner = () => {
+        BinarySocket.wait('website_status', 'authorize', 'landing_company').then(() => {
+            if (!isEuCountry()) {
+                const $binary_grid_banner = $('#binary_grid_banner');
```

Comment:
better use javascript since we're just setting visibility. so `getElementById('binary_grid_banner').setVisibility(1);` (feel free to look up the `getElementById` function versus the `document.getElementById`)